### PR TITLE
Updated research-community-forum-3-june-2023.md

### PR DESCRIPTION
**Pull request by laurenwigmore**
Updated blog post to include the research call video and an update on what was covered in the call.

### CODE REVIEW

File: content/blog/research-community-forum-3-june-2023.md [hunk 1]
```diff
@@ -7,14 +7,11 @@ authors:
   - rufuspollock
 image: /assets/images/research-forum-3.jpg
 ---
+>This video forms part of our series of [[/research|research community]] monthly fora.
 
-Jonathan Reams will present at the third [[/research/|research community forum]]. Jonathan is Associate Professor at the Norwegian University of Science and Technology and has a wealth of expertise spanning many areas across adult and leadership development, consciousness research and transformative learning. Jonathan practices the cultivation of leadership through consulting and leadership development program design and delivery. He brings awareness based technology to this work, focusing on how the inner workings of human nature can develop leadership capacities for today’s complex challenges. He is one of the leading experts in Europe on [Lectica](https://lecticalive.org), one of the main frameworks for adult developmental assessment.
```
Comment (rufuspollock) [inline content/blog/research-community-forum-3-june-2023.md:11]: @laurenwigmore we wouldn't remove this info - we'd just add the new info at the bottom of the post in a section called e.g. "Post event info"

File: content/blog/research-community-forum-3-june-2023.md [hunk 2]
```diff
@@ -7,14 +7,11 @@ authors:
   - rufuspollock
 image: /assets/images/research-forum-3.jpg
 ---
+>This video forms part of our series of [[/research|research community]] monthly fora.
```
Comment (rufuspollock) [inline content/blog/research-community-forum-3-june-2023.md:10]: Why are we blockquoting this? no need ...

File: content/blog/research-community-forum-3-june-2023.md (hunk 1 above)
Comment (rufuspollock) [reply content/blog/research-community-forum-3-june-2023.md:11]: OK, i now see you've just split things apart, not removed stuff. That makes sense!

File: content/blog/research-community-forum-3-june-2023.md [hunk 3]
```diff
@@ -7,14 +7,11 @@ authors:
...
+https://youtu.be/uka-m35-_gA
 
-Jonathan will discuss the journey so far for Lectica over the past 20 years, along with his experience in applying Lectica, and the future vision of the application of developmental assessments, before opening out the call into a wider discussion.
+In the third of the Life Itself Research calls Jonathan Reams covers personal development, understanding growth across domains, intelligence testing, skill correlation, self-mastery, learning, emotions, and the challenges and practical applications of developmental theories. 
 
-**When:** 15:00 European, Friday June 9th, 2023
```
Comment (rufuspollock) [inline content/blog/research-community-forum-3-june-2023.md:15]: I'd keep this info even if it is now obsolete. Not worth removing it.

File: content/blog/research-community-forum-3-june-2023.md [hunk 4]
```diff
@@ -7,14 +7,11 @@ authors:
...
+https://youtu.be/uka-m35-_gA
 
-Jonathan will discuss the journey so far for Lectica over the past 20 years, along with his experience in applying Lectica, and the future vision of the application of developmental assessments, before opening out the call into a wider discussion.
+In the third of the Life Itself Research calls Jonathan Reams covers personal development, understanding growth across domains, intelligence testing, skill correlation, self-mastery, learning, emotions, and the challenges and practical applications of developmental theories.
```
Comment (rufuspollock) [inline content/blog/research-community-forum-3-june-2023.md:14]: Nice summary 👏

**Review by rufuspollock — APPROVED** [review]
Again i think easiest to merge and we can take the feedback for next time 😄

**Comment by laurenwigmore** [comment]
@rufuspollock thanks Rufus. Quick Q do I need to delete the branch? 

"Pull request successfully merged and closed
You’re all set—the laurenwigmore-patch-5 branch can be safely deleted."?

**Pull request merged**